styles/app.scss

```diff
@@ -4,3 +4,294 @@
+  text-align: center;
+
+  &:hover {
+    color: $brand_white;
+    background: $brand-primary;
+  }
+}
+
+
+.shirt-div {
+  width: 100%;
+  padding-top: $gutter;
+  padding-right: $gutter;
+  padding-left: $gutter;
+
+}
+
+img {
+  width: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+.shop-item-title {
+  text-align: center;
+  color: $brand-primary;
+  padding-top: $gutter;
+  padding-right: $gutter;
+  padding-left: $gutter;
+
+}
+
+.shop-item-description {
+  color: $brand-primary;
+  text-align: center;
+}
+
+.shop-button {
+  list-style: none;
+}
+
+.dropdown {
+  position: relative;
+}
+
+.fa {
+  font-size: 1.9rem;
+  padding: 0.3rem;
+
+  &:hover {
+    font-size: 2.1rem;
+  }
+}
+
+.shirt-size {
+  font-size: 1.9rem;
+  text-align: center;
+}
+
+.dropdown__options {
+  display: none;
+  position: absolute;
+  text-align: center;
+  padding: 0;
+  width: 100%;
+  bottom: 100%;
+
+  &-item-shirt-color-pink {
+    width: 100%;
+    color: $pink;
+    border: 0.2rem solid $brand-primary;
+
+    &:hover {
+      font-size: 2rem;
+    }
+
+  }
+
+  &-item-shirt-color-blue {
+    color: $blue;
+    border: 0.2rem solid $brand-primary;
+
+    &:hover {
+      font-size: 2rem;
+    }
+
+  }
+
+  &-item-shirt-color-yellow {
+    color: $yellow;
+    border: 0.2rem solid $brand-primary;
+
+    &:hover {
+      font-size: 2rem;
+    }
+
+  }
+
+  &-item-shirt-color-green {
+    color: $green;
+    border: 0.2rem solid $brand-primary;
+
+    &:hover {
+      font-size: 2rem;
+    }
+
+  }
+
+  &-item {
+    border: 0.2rem solid $brand-primary;
+  }
+}
+
+
+.dropdown:hover .dropdown__options {
+  display: flex;
+  flex-direction: column;
+  background: $brand-white;
+
+  position: absolute;
+
+}
+
+
+.dropdown__options-item {
+  text-align: center;
+  font-size: 1.5rem;
+  padding-top: 0.3rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  // width: 100%;
+
+  &:hover {
+    font-size: 1.7rem;
+  }
+}
+
+.dropdown__options {
+  color: $brand_primary;
+  background: $brand-white;
+}
+
+@media screen and (min-width: 500px) and (max-width: 980px) {
+  .container {
+    max-width: 500px;
+    margin: 0 auto;
+  }
+
+  .row {
+    display: block;
+    flex-flow: column;
+  }
+
+  .grid-item {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+  }
+
+  .shop-item {
+    display: flex;
+  }
+
+  .shirt-div {
+    padding: $gutter;
+  }
+
+  .shop-item-buttons {
+    flex-direction: column;
+    justify-content: space-between;
+    width: 10%;
+    border-top: 0;
+    border-left: 0.3rem solid $brand-primary;
+
+  }
+
+  .shop-button {
+    width: 100%;
+    border-bottom: none;
+    border-right: none;
+    border-left: none;
+    border-top: 0.3rem solid $brand-primary;
+    flex-grow: 1;
+
+      &:nth-child(1) {
+        border-top: none;
+      }
+  }
+
+
+  .dropdown:hover .dropdown__options {
+    flex-direction: row;
+    background: $brand_white;
+  }
+
+
+
+  .dropdown__options {
+    position: absolute;
+    right: 412%;
```

Comment:
If you set `width: auto` in this media query to reset your `width: 100%` you'll be able to use a sane number here. The bottom value can also be modified to make it centered by pairing it with a `transform: translateY`